src/components/forms/pool_actions/MigrateForm/constants.ts

```diff
@@ -0,0 +1,21 @@
+import { keyBy } from 'lodash';
+
+import { configService } from '@/services/config/config.service';
+
+import { PoolMigrationType, PoolMigrationInfo } from './types';
+
+const riskI18nLabelPrefix = 'migratePool.previewModal.riskWarnings.risks';
+
+export const POOL_MIGRATIONS: PoolMigrationInfo[] = [
```

Comment:
The idea here is to create valid migrations (`fromPoolId` and `toPoolId`) - its easy to tweak it later (for example perhaps there could be multiple `toPoolIds`)